Etabs_Adapter/CRUD/Read/Results/MeshResults.cs

```diff
@@ -481,7 +669,9 @@ private List<MeshForce> SmoothenForces(List<MeshForce> forces)
                 double vx = group.Average(x => x.VX);
                 double vy = group.Average(x => x.VY);
 
-                smoothenedForces.Add(new MeshForce(first.ObjectId, first.NodeId, "", first.ResultCase, first.ModeNumber, first.TimeStep, first.MeshResultLayer, first.LayerPosition, MeshResultSmoothingType.ByPanel, first.Orientation, nxx, nyy, nxy, mxx, myy, mxy, vx, vy));
+                smoothenedForces.Add(new MeshForce(first.ObjectId, first.NodeId, "", first.ResultCase, first.ModeNumber, first.TimeStep, 
+                                                        first.MeshResultLayer, first.LayerPosition, MeshResultSmoothingType.ByPanel, first.Orientation, 
+                                                        nxx, nyy, nxy, mxx, myy, mxy, vx, vy));
```

Comment:
Why have these been modified? This is going to confuse people who look back on this in a few years wondering what changes you made to `MeshForce` either via the PR or Blame.

Reply:
@peterjamesnugent, looks to me here just the indentation of the code has changed. Would you prefer me to turn code lines alignment to their original?

Reply:
@peterjamesnugent, code lines restored as per they original setting out ;)